features/advocate_new_claim.feature

```diff
@@ -181,3 +181,24 @@ Feature: Advocate new claim
       And I submit to LAA
      Then There should not be any Initial Fees saved
       And There should be a Miscellaneous Fee Saved
+
+  @javascript @webmock_allow_localhost_connect
```

Comment:
This breaks my test - with:
```
 Real HTTP connections are disabled. Unregistered request: GET http://127.0.0.1:58747/__identify__ with headers {'Accept'=>'*/*', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'User-Agent'=>'Ruby'}
```
i believe because the JS assets are not available - may be a setup issue??